### PR TITLE
Enhance API Reference - Documentation improvements

### PR DESCRIPTION
**Pull request by gtsonevv**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/near/near-sdk-js/blob/master/CONTRIBUTING.md).
- [x] Commit messages follow the [conventional commits](https://www.conventionalcommits.org/) spec
- [ ] **If this is a code change**: I have written unit tests.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### CODE REVIEW

File: packages/near-contract-standards/src/fungible_token/core_impl.ts
Comment (frol) [inline packages/near-contract-standards/src/fungible_token/core_impl.ts:126]: [FT standard](https://github.com/near/NEPs/blob/master/neps/nep-0141.md#ft_on_transfer) defined the encoding for the input/outputs as JSON. Since JSON does not support big integers, the FT standard uses a string type to pass the unused_amount. It would be cleaner to use `JSON.parse()` instead of manually "parsing" quotes out.

File: packages/near-contract-standards/src/fungible_token/receiver.ts
Comment (frol) [inline packages/near-contract-standards/src/fungible_token/receiver.ts]: Formatting is broken here, and in a few other places in the doc string examples. Please, fix them

File: packages/near-contract-standards/src/fungible_token/resolver.ts
Comment (frol) [inline packages/near-contract-standards/src/fungible_token/resolver.ts]: Formatting issue

**Review by frol — CHANGES_REQUESTED** [review]
@gtsonevv

File: packages/near-contract-standards/src/fungible_token/core_impl.ts
Comment (gtsonevv) [reply packages/near-contract-standards/src/fungible_token/core_impl.ts:126]: @frol, this is out of this PR's scope, would be better in a separate PR

**Review by frol — APPROVED** [review]
@race-of-sloths score 5

**Comment by race-of-sloths** [comment]
@gtsonevv Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!


[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/gtsonevv?type=bot&pr=near/near-sdk-js/410&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/gtsonevv?type=bot&pr=near/near-sdk-js/410&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/gtsonevv)




<details><summary>Current status: <i>executed</i></summary>

| Reviewer | Score |
|--------|--------|
| @frol  | 5 |


Your contribution is much appreciated with a final score of 5!
You have received 50 Sloth points for this contribution



</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/gtsonevv)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Pull request merged**
